ui/src/override/components/flows/blueprints/BlueprintsLayout.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+    <div class="blueprints">
+        <slot />
+    </div>
+</template>
+
+<style scoped lang="scss">
+    @import "@kestra-io/ui-libs/src/scss/variables.scss";
+
+    .blueprints {
+        background: var(--bg-dots-image) no-repeat top left;
+        background-color: var(--bg-dots-color);
+        background-size: 1654px;
```

Comment:
More of a thinking out loud then anything else, but do we want to hard code the size in pixels?